javascript-loops-practice/filter-out-strings.js

```diff
@@ -1,13 +1,9 @@
 /* exported filterOutStrings */
 function filterOutStrings(values) {
   var newArray = [];
-  var i = values.length;
-  while (i > 0) {
-    if (typeof (values[i - 1]) === typeof (7)) {
-      newArray.unshift(values[i - 1]);
-      i--;
-    } else {
-      i--;
+  for (var i = 0; i < values.length; i++) {
+    if (typeof (values[i]) === typeof (7)) {
+      newArray.push(values[i]);
     }
```

Comment:
This will just catch values that are numbers, not values that aren't strings. If the tests included other data types as well, this wouldn't work.